app/views/documents/index.html.erb

```diff
@@ -1,32 +1,30 @@
 <% content_for :title, t("documents.index.title") %>
 
-<%= render "govuk_publishing_components/components/button", {
+<% content_for :title_side, render("govuk_publishing_components/components/button", {
   text: "New document", href: new_document_path, margin_bottom: true
-} %>
+}) %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-one-third">
     <%= render "documents/index/filters" %>
 
-    <br/>
-    <hr/>
-    <br/>
+    <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--l">
 
     <div class="govuk-body">
-      <strong> <%= t("documents.index.guidance.title") %></strong>
-      <br/>
-      <%= link_to "Planning, writing and managing content",
-        "https://www.gov.uk/topic/government-digital-guidance/content-publishing", class: "govuk-link" %>
+      <p class="govuk-heading-s"> <%= t("documents.index.guidance.title") %></p>
+      <p><%= link_to "Planning, writing and managing content", "https://www.gov.uk/topic/government-digital-guidance/content-publishing", class: "govuk-link" %></p>
```

Comment:
Think we're missing 'target: _blank' here ;-).

Reply:
Enjoy your holiday Ben!